src/xcode/ENA/ENA/Source/Scenes/TraceLocations/TraceLocationsCoordinator.swift

```diff
@@ -97,6 +97,7 @@ class TraceLocationsCoordinator {
 	}()
 	
 	private func showInfoScreen() {
+		let alreadyDidConsentOnce = infoScreenShown == true
```

Comment:
```suggestion
		let alreadyDidConsentOnce = infoScreenShown
```
I don't get why the `== true` is needed here

Reply:
I just think the code is easier to read that way

Reply:
So `alreadyDidConsentOnce`  gets `true` if `infoScreenShown == true` and otherwise `false` - only depending on current value of `infoScreenShown`
For me it's not easier to read.